CHANGELOG.md

```diff
@@ -6,11 +6,11 @@ This document tracks pending changes to packages. It is facilitating the write-u
 
 | Package                                    | Released version   | Target version |
 |--------------------------------------------|--------------------|----------------|
-| ContingentClaims.Core                      | 2.0.0              | unchanged      |
```

Comment:
I think that for contingent claims a patch update should be enough, what do you think?

Reply:
Agree